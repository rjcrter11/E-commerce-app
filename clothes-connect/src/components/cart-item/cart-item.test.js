import React from 'react';
import { create } from 'react-test-renderer';
import CartItem from './cart-item.component';


it('should render CartItem component', () => {
    const mockItem = {
        imageUrl: 'www.testImg.com',
        price: 10,
        name: 'hats',
        quantity: 2
    }
    const wrapper = create(<CartItem item={mockItem} />)
    expect(wrapper.toJSON()).toMatchSnapshot()
});