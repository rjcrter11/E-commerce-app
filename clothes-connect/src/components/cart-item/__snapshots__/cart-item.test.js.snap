// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartItem component 1`] = `
<div
  className="cart-item"
>
  <img
    alt="item"
    src="www.testImg.com"
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    >
      hats
    </span>
    <span
      className="price"
    >
      2
       x $
      10
    </span>
  </div>
</div>
`;
